src/app/pages/AddCard/AddCard.tsx

```diff
@@ -42,12 +42,22 @@ export default function AddCard(): JSX.Element {
       children: collection.name,
       language: collection.language,
       onClick: () => addToCard(collection),
+      //Here is Error
+      active: selectedCollections.find(
+        (element) =>
+          element ===
+          {
+            children: collection.name,
+            language: collection.language,
+          }
```

Comment:
You can't compare objects like this.
Equality operator in objects compares by reference. This means, it checks if the object reference in memory is the same for both the left and the right side of the equality operator.

In this case, you should compare the properties of the objects, one by one.

Check this little article (I didn't read it, but it looks fine)
https://dmitripavlutin.com/how-to-compare-objects-in-javascript/

Reply:
Thank you very much :D 
Problem solved